sympy/series/formal.py

```diff
@@ -1143,6 +1143,66 @@ def integrate(self, x=None, **kwargs):
 
         return self.func(f, self.x, self.x0, self.dir, (ak, self.xk, ind))
 
+    def convolve(self, other, x=None, order=4):
```

Comment:
I believe `order` shouldn't hold a default value. It should be totally customized and user should be given an error message in case he doesn't passes the order.

Reply:
Hi @czgdp1807 ,

This is something in line with how series or other functions work. 

Hi @ArighnaIITG ,

Can you make default 6 for now?

Reply:
Please rename ``order`` to ``n``. It seems we use ``n`` in other functions like ``fps.truncate``, ``series`` method.

Reply:
Thanks for the clarification @leosartaj. Please resolve this conversation.

Reply:
Sure @leosartaj